base-notebook/start.sh

```diff
@@ -53,6 +53,15 @@ unset_explicit_env_vars () {
     fi
 }
 
+get_user_id () {
+    id -u "${1}" 2>/dev/null
+    return 0
+}
+
+get_group_id () {
+    id -g "${1}" 2>/dev/null
+    return 0
+}
```

Comment:
Hmm, doesn't this always return 0, no matter what?

Note that when I look at this PR, I don't yet understand the situation you are in etc so I'm struggling with the review to conclude if it would make sense.

From a technical standpoint without understanding the intent of the changes, the function name above doesn't seem to do what its name indicate. It seems to include a call to the `id` command, but ignore its output - so then I'd wonder why even make the call in the first place for example.

I hope its okay that I ask you to open an issue to focus on what you perceive as a bug as a first priority. If it is a bug, the second step would be on how to fix it. Right now, I'm not yet understanding the bug, but figure there is some issue with an implementation to fix it - but the implementation won't be relevant before we can agree on a bug exists and should be fixed.

Due to that, I'll go for a close and hope you feel alright with opening an issue to help us reach agreement there is a bug to fix first.

Reply:
If you have multiple separate bugs or things you suggest we change, it is probably helpful if they are separate github issues focused one specific change.